src/services/proposals.ts

```diff
@@ -108,3 +109,34 @@ export async function getProposalFromDb(mySql: ServerlessMysql, proposalId: stri
 
   return { dbProposal, dbProposalPasswordData };
 }
+
+export interface IUpdateProposalDbInputs {
+  partialTx: string,
+  signatures?: string,
+}
+
+export async function updateProposalOnDb(mySql: ServerlessMysql, current: IDbProposal, updateData: IUpdateProposalDbInputs) {
+  const proposalId = current.id;
+
+  // Checking if there was a change on the proposal data, or only on its signatures
+  let newHistory: IDbProposal['history'] | undefined;
+  if (current.partialTx !== updateData.partialTx) {
+    newHistory = [
+      { partialTx: current.partialTx, timestamp: current.timestamp },
+      ...current.history,
+    ].slice(0, MAX_HISTORY_LENGTH);
+  }
+
+  const mySqlQuery = `UPDATE proposals
```

Comment:
Lint rule `operator-linebreak` doesn't allow to break this line here, so the query indentation got a bit wide.